views/default/resources/admin/plugin_screenshot.img.php

```diff
@@ -9,11 +9,11 @@
 
 $plugin = elgg_get_plugin_from_id($plugin_id);
 if (!$plugin) {
-	$file = elgg_get_root_path() . '_graphics/icons/default/medium.png';
+	$file = elgg_get_root_path() . 'views/default/icons/default/medium.png';
```

Comment:
Plugin screenshot needs the file path, not the URL. I suppose I could use the url and let PHP's file handling of URLs kick in, but I didn't think of it at the time and this seemed like an easier refactoring.